src/cli.rs

```diff
@@ -0,0 +1,19 @@
+use clap::Parser;
+use std::path::PathBuf;
+
+#[derive(Parser, Debug)]
+#[clap(version, about, long_about = None)]
+pub struct CliArgs {
+    #[clap(short, long)]
+    pub config_path: Option<PathBuf>,
+}
+
+pub enum CliCommand {
```

Comment:
Why is this called `Command`? Does it "command" something, or emits a CLI command?
Bad naming